### ISSUE TITLE
Bug in Commandline using Tablet

### ISSUE THREAD

**Issue by AcroDirki** [OpenAero/main#223]
Hi guys,

When I use the OA-App on my Samsung Galaxy Tab 8" with Android and I type into the command line 

id2f,if

then 

idid2id2f

appears. This happens indepently of the figure for most figure strings? Is this caused by a autofill function from android or indeed a bug in OA?

Kind regards,
D.

**Comment by OpenAero** [comment]
Could not reproduce on Samsung A5 Android mobile. Any chance to record screen video of this happening?

**Comment by AcroDirki** [comment]
Well, I meanwhile identified that this occurs (or not) pending on the phone (samsung keyboard) settings. If there is an easy opportunity within OA to suppress any "Autocorrect" or similar functions coming alone with the diverse mobile devices in the string field, this should be done, otherwise we have to live with it as I assume that any detailed solution for every device is just out of scope...

**Comment by OpenAero** [comment]
At present it can not be done in the browser. It might be possible on the iOS/Android app and may also become possible in the browser in the future. Of course it is possible for the user to turn off their autocorrect function themselves.

**Comment by AcroDirki** [comment]
Agree.
However, it would be a "nice to have", if OA would do so automatically to assist the "lazy user".

**Comment by OpenAero** [comment]
Fixed
